src/main/java/org/folio/dew/service/JobCommandsReceiverService.java

```diff
@@ -104,42 +104,49 @@ public void postConstruct() {
   }
 
   @KafkaListener(
-    id = KafkaService.EVENT_LISTENER_ID,
+    id = EVENT_LISTENER_ID,
     containerFactory = "kafkaListenerContainerFactory",
     topicPattern = "${application.kafka.topic-pattern}",
     groupId = "${application.kafka.group-id}")
   public void receiveStartJobCommand(JobCommand jobCommand, Acknowledgment acknowledgment) {
     log.info("Received {}.", jobCommand);
 
-    try {
-      if (deleteOldFiles(jobCommand, acknowledgment)) {
-        return;
-      }
-      log.info("-----------------------------JOB---STARTS-----------------------------");
-
-      prepareJobParameters(jobCommand);
+    kafkaManager.pauseConsume(EVENT_LISTENER_ID);
+    CompletableFuture.runAsync(() -> launchJob(jobCommand, acknowledgment))
+      .handle((unused, throwable) -> {
+        if (throwable != null) log.error(throwable.toString(), throwable);
+        kafkaManager.resumeConsumer(EVENT_LISTENER_ID);
+        return null;
+      });
```

Comment:
So this makes it not possible to process more events when some long-run job is in progress. Am I right?